### ISSUE TITLE
Critical: Cannot reproduce training - missing annotations in 40K subset and missing data processing module

### ISSUE THREAD

**Issue by fjzsy** [ArchiAI-LAB/ArchCAD#8]
Dear authors,

We are trying to reproduce the DPSS results on ArchCAD-400k as described in your NeurIPS 2025 paper. However, we've encountered several critical issues that prevent us from running the training code. We'd like to report these findings and seek your guidance.

Environment
Hardware: RTX 5060 Ti

Software: PyTorch 2.8.0, CUDA 12.8, Python 3.9

OS: Ubuntu 22.04 (WSL2 on Windows 11)

Issues Found
1. Missing data processing module (svgnet.data)
The code imports from svgnet.data, but this module is not present in the repository. The modules/ directory contains only point cloud utilities, and no data loading code is provided.

2. Pointops compilation failures
The project depends on an outdated version of pointops that uses THC/THC.h, which has been removed in PyTorch 2.x. We had to manually modify the source code to get it to compile. This could be avoided by using a more modern point cloud library.

3. Critical: Empty annotations in the released subset
The released 40K subset contains no labels. All JSON files we checked have empty semantic_labels and instance_labels:

text
检查样本: 82eeb1f9-3331-4460-b19e-7728ccf5f7ea
  semantic_labels: 0 个
  instance_labels: 0 个

检查样本: 01aedcee-2deb-4eb6-b8e2-397949109890
  semantic_labels: 0 个
  instance_labels: 0 个

检查样本: f91cbf4e-fc6a-4c13-bac6-a64001b76802
  semantic_labels: 0 个
  instance_labels: 0 个

=== 整体统计 ===
JSON 文件总数: 41097
前100个中空标签文件数: 100
Without labels, training is impossible.

4. Missing preprocessing tools
The README mentions CADParser will be released, but currently there's no way to process raw SVG files into the format expected by the model.

Our Goal
We are trying to validate your paper's claims by reproducing the experiments. We're not just trying to run the code - we want to verify that the reported 70.6% PQ is achievable.

Questions
Are the labels for the 40K subset supposed to be included? If so, how can we access them?

Is there a plan to release the data processing code (CADParser)?

Could you provide a Docker image with a working environment?

What We've Done
Successfully compiled pointops after manual fixes

Implemented missing data loading modules

Set up the environment with all dependencies

Verified data integrity (files exist, but labels missing)

We appreciate your work and hope to collaborate in validating the results. Any guidance would be greatly appreciated.
